src/_pytest/fixtures.py

```diff
@@ -1666,6 +1706,7 @@ def _register_fixture(
             params=params,
             ids=ids,
             _ispytest=True,
+            _autouse=autouse,
```

Comment:
I'm surprise `FixtureDef` didn't already have an attribute marking it as autouse... :eyes: